corehq/apps/data_analytics/tasks.py

```diff
@@ -34,3 +34,11 @@ def _last_month_datespan():
         message,
         text_content=message
     )
+
+
+@periodic_task(queue='background_queue', run_every=crontab(hour=2, minute=0, day_of_week='*'),
+               ignore_result=True)
```

Comment:
actually one thing I should mention is that this creates a task that runs the malt every day. i ran it on prod in the middle of the day today and it took about 30 minutes and didn't have any noticeable affect (ES CPU went up to 50% for a bit and then came back down). I think we can revisit the frequency if we notice any side effects.